### PR TITLE
Avoid wrapping the 12 bytes in yet another object

### PR DESCRIPTION
**Pull request by baldersheim**
@bratseth PR

### CODE REVIEW

File: container-search/src/main/java/com/yahoo/prelude/fastsearch/FS4FillInvoker.java [hunk 1]
```diff
@@ -132,10 +125,11 @@ public void release() {
         }
     }
 
-    private int countFastHits(Result result) {
+    private int countFastHits(Result result, String summaryClass) {
```
Comment (bratseth) [inline container-search/src/main/java/com/yahoo/prelude/fastsearch/FS4FillInvoker.java]: Should change name to countUnfilledFastHits

File: container-search/src/main/java/com/yahoo/prelude/fastsearch/FastHit.java [hunk 2]
```diff
@@ -41,7 +39,7 @@
 
     /** The global id of this document in the backend node which produced it */
     //Todo should be bytearray directly and generate GlobvalId on access
```
Comment (bratseth) [inline container-search/src/main/java/com/yahoo/prelude/fastsearch/FastHit.java]: Remove TODO

**Review by bratseth — APPROVED** [review]

File: container-search/src/main/java/com/yahoo/prelude/fastsearch/FS4FillInvoker.java (hunk 1 above)
Comment (baldersheim) [reply container-search/src/main/java/com/yahoo/prelude/fastsearch/FS4FillInvoker.java]: Fixed

File: container-search/src/main/java/com/yahoo/prelude/fastsearch/FastHit.java (hunk 2 above)
Comment (baldersheim) [reply container-search/src/main/java/com/yahoo/prelude/fastsearch/FastHit.java]: Fixed

**Pull request merged**